infrastructure/restapi/src/main/java/tech/pegasys/teku/infrastructure/restapi/endpoints/EndpointMetadata.java

```diff
@@ -87,6 +105,8 @@ public void writeOpenApi(final JsonGenerator gen) throws IOException {
     gen.writeStringField("summary", summary);
     gen.writeStringField("description", description);
 
+    OpenApiRequestBody.writeOpenApi(gen, requestBodyType);
```

Comment:
Ah meant to come back to this comment.  Given that we didn't wind up testing the `writeOpenApi` method separately, I think I'd make it just be a private method here rather than a static in another class.